### PR TITLE
Fix and div

### PR DESCRIPTION
**Pull request by tr211**

### CODE REVIEW

File: ps_store_repository.py [hunk 1]
```diff
@@ -7,12 +7,17 @@ class Game(BaseModel):
     country: str
     title: str
     currency: str
-    price: float
+    price: str
```
Comment (radprofy) [inline ps_store_repository.py:10]: We still need price as float.

File: ps_store_repository.py [hunk 2]
```diff
@@ -7,12 +7,17 @@ class Game(BaseModel):
...
     publisher: str
 
 def find_ps4_prices(game: str, countries_dict: dict)->list:
     game_html_format = game.replace(' ','-')
     search_game = game_html_format.lower()
+    apostrophe = ["'", "®"]
+    for symbol in apostrophe:
+        search_game = search_game.replace(symbol, '')
```
Comment (radprofy) [inline ps_store_repository.py:18]: You can do that in line 14.

File: fastAPI.py [hunk 3]
```diff
@@ -13,9 +14,11 @@ def country_prices(game_name)->list[ps_store_repository.Game]:
     return list_of_prices
 
 @app.get('/ps4-names')
-def ps4_games_names()->list[str]:
+def ps4_games_names() -> list[str]:
     names_list = names_registry.get_all_ps4_games()
-    return names_list 
-
+    if isinstance(names_list, list):
```
Comment (radprofy) [inline fastAPI.py:19]: I don't think we should move that here.

File: ps_store_repository.py (hunk 1 above)
Comment (radprofy) [reply ps_store_repository.py]: test

Comment (tr211) [reply ps_store_repository.py]: try

File: fastAPI.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import re
```
[the remark this answers is not in the document]
Comment (tr211) [reply fastAPI.py:1]: removet

**Pull request merged**